tests/protodata-extension/src/main/java/io/spine/protodata/test/UuidJavaRenderer.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2021, TeamDev. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.protodata.test;
+
+import com.google.common.collect.ImmutableList;
+import io.spine.protodata.FilePath;
+import io.spine.protodata.TypeName;
+import io.spine.protodata.codegen.java.ClassName;
+import io.spine.protodata.codegen.java.JavaRenderer;
+import io.spine.protodata.renderer.InsertionPoint;
+import io.spine.protodata.renderer.SourceSet;
+
+import java.nio.file.Path;
+import java.util.Set;
+import java.util.UUID;
+
+import static java.lang.String.format;
+import static java.lang.System.lineSeparator;
+
+/**
+ * A renderer which adds the {@code randomId()} factory methods to the UUID types.
+ *
+ * <p>A UUID type is a message which only has one field — a {@code string} field called
+ * {@code uuid}.
```

Comment:
Let's invite "called" from the previous line to spend some quality time on this line along with `uuid`.